css/style.css

```diff
@@ -165,15 +173,44 @@ a {
 .headline h1 {
   font-size: 2.2rem;
   line-height: 1.5;
-  color: #b93223;
   margin-bottom: 1rem;
   font-weight: bolder;
+  font-family: "COCOGOOSE", sans-serif;
+  color: transparent;
+  background-image: url("../images/fire-bg.jpg");
+  background-size: 200%;
+  background-position: right top;
+  background-clip: content-box;
+  -webkit-background-clip: text;
+  transition: 5s ease-out;
+}
+
+.welcome-text h1 {
+  font-size: 1.8rem;
+  line-height: 1.5;
+  color: #b93223;
+  text-align: center;
+  margin-bottom: 1rem;
+  font-weight: 700;
+  font-family: "COCOGOOSE", sans-serif;
+}
+
+.headline h1:hover {
+  background-position: left bottom;
 }
 
 .headline-box {
   width: 100%;
   padding: 1rem;
   border: 2px solid #fff;
+  transition: 0.5s ease;
+}
+
+.headline-box:hover {
+  border: 2px solid #ec5242;
+  transform: translateY(-0.8rem);
```

Comment:
- [ ] [OPTIONAL] It looks like your about page has some issues on the mobile screens where some texts in the headline overlap each other. Not only in that part but also in the footer section. Please try to correct that because it may discourage user experience on your page. Take a look at the screenshots I have taken 👇 
![image](https://user-images.githubusercontent.com/90222110/181594777-0ef5e933-4172-4778-81c2-9bd8c6a5d552.png)


![image](https://user-images.githubusercontent.com/90222110/181594962-4e075b19-e235-4f7d-a35e-984ba3d69691.png)


![image](https://user-images.githubusercontent.com/90222110/181595129-26ad208a-0d83-4351-8666-b9e823813e8d.png)

Reply:
Thank you @BertrandConxy  for your professional review. I appreciate you bringing this to my notice. i have made some corrections and improvement.